Apps/Sandcastle/gallery/Cesium OSM Buildings Styling.html

```diff
@@ -0,0 +1,260 @@
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script
+      type="text/javascript"
+      src="../../../Build/CesiumUnminified/Cesium.js"
+      nomodule
+    ></script>
+    <script type="module" src="../load-cesium-es6.js"></script>
+  </head>
+
+  <body
+    class="sandcastle-loading"
+    data-sandcastle-bucket="bucket-requirejs.html"
+  >
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay">
+      <h1>Loading...</h1>
+    </div>
+    <div id="toolbar"></div>
+    <script id="cesium_sandcastle_script">
+      function startup(Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        var viewer = new Cesium.Viewer("cesiumContainer", {
+          terrainProvider: Cesium.createWorldTerrain(),
+        });
+
+        // Create the OSM buildings
+        var buildings = Cesium.createOsmBuildings();
+        viewer.scene.primitives.add(buildings);
+
+        // Set the initial camera to look at Hong Kong
+        viewer.scene.camera.flyTo({
+          destination: Cesium.Cartesian3.fromDegrees(-122.3472, 47.598, 370),
+          orientation: {
+            heading: Cesium.Math.toRadians(10),
+            pitch: Cesium.Math.toRadians(-10),
+          },
+        });
+
+        // Styling functions
+        function colorByHeight() {
+          buildings.style = new Cesium.Cesium3DTileStyle({
+            color: {
+              conditions: [
+                [
+                  "${feature['cesium#estimatedHeight']} >= 100",
+                  "color('#581845')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 50",
+                  "color('#900C3F')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 25",
+                  "color('#C70039')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 10",
+                  "color('#FF5733')",
+                ],
+                ["true", "color('#FFC300')"],
+              ],
+            },
+          });
+        }
+
+        function colorByLatitude() {
+          buildings.style = new Cesium.Cesium3DTileStyle({
+            defines: {
+              lagitudeRadians: "radians(${feature['cesium#latitude']})",
+            },
+            color: {
+              conditions: [
+                ["${lagitudeRadians} >= 0.83117", "color('purple')"],
+                ["${lagitudeRadians} >= 0.83112", "color('red')"],
+                ["${lagitudeRadians} >= 0.83107", "color('orange')"],
+                ["${lagitudeRadians} >= 0.83102", "color('yellow')"],
+                ["${lagitudeRadians} >= 0.83097", "color('lime')"],
+                ["${lagitudeRadians} >= 0.83092", "color('cyan')"],
+                ["true", "color('blue')"],
+              ],
+            },
+          });
+        }
```

Comment:
I can't think of a realistic use case for styling by just latitude, other than maybe debugging. I'd remove this, since styling by distance is more useful/shows you how to use the lat/lon properties.